src/main/resources/changesets/changelog-consent-49.0.xml

```diff
@@ -5,25 +5,26 @@
     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet author="grushton" id="49.0">
+        <validCheckSum>ANY</validCheckSum>
         <addColumn tableName="dacuser">
             <column name="status" type="int" defaultValue="0"/>
             <column name="rationale" type="text"/>
         </addColumn>
-        <sql>
-            update dacuser set status = 0 where dacUserId in (select user_id from user_role where status = 0 and role_id = 5)
+        <sql dbms="hsqldb, mysql">
```

Comment:
again here and below: do you still need dbms="hsql, mysql" here?

Reply:
Yes, to restrict them from failing in postgres.